src/server/publicApi.js

```diff
@@ -113,6 +113,69 @@ const runPublicApi = (gameServer) => {
     ctx.body = model;
     });
 
+    //produce a nice textfile with the threats in
+    router.get('/download/text/:id', async (ctx) => {
+        //get some variables that might be useful
+        const gameName = ElevationOfPrivilege.name;
+        const gameID = ctx.params.id;
+        const res = await gameServer.db.get(`${gameName}:${gameID}`);
+        const metadata = await gameServer.db.get(`${gameName}:${gameID}:metadata`);
+        const model = await gameServer.db.get(`${gameName}:${gameID}:model`);
+        var threats = [];
```

Comment:
One could think about refactoring this whole code block into a function `getThreats` or so...